seedlet/classes/class-seedlet-custom-colors.php

```diff
@@ -14,19 +14,35 @@
 class Seedlet_Custom_Colors {
 
 	private $seedlet_custom_color_variables = array();
+	private $default_background             = '#FFFFFF';
+	private $default_foreground             = '#333333';
+	private $default_primary                = '#000000';
+	private $default_secondary              = '#3C8067';
+	private $default_tertiary               = '#FAFBF6';
+	private $default_border                 = '#EFEFEF';
 
-	function __construct() {
+	function __construct( $background, $foreground, $primary, $secondary, $tertiary, $border ) {
```

Comment:
I think this needs to be:

```suggestion
	function __construct( $background = $default_background, $foreground = $default_foreground, $primary = $default_primary, $secondary = $default_secondary, $tertiary = $default_tertiary, $border = $default_border ) {
```